en/docs/administer/organization.md

```diff
@@ -20,7 +20,6 @@ Choreo roles are defined as follows:
 - **API Publisher**: An API Publisher can discover, create, publish, delete, test, and manage an API.
 
 - **API Subscriber**:  An API Subscriber is a developer in a particular organization. An API subscriber can subscribe to an application, manage subscriptions, manage applications, generate API keys, and manage API keys.
```

Comment:
I suppose we can use sentence style for the terms "API publisher" and "API subscriber". We seem to have used the title style in some places.

Reply:
Fixed.